src/components/Header/Header.tsx

```diff
@@ -43,7 +42,9 @@ const FeedbackRoute = ({ user }: { user: DeepRequired<ChromeUser> }) => {
 export const Header = ({ breadcrumbsProps }: { breadcrumbsProps?: Breadcrumbsprops }) => {
   const searchEnabled = useFlag('platform.chrome.search.enabled');
   const user = useSelector(({ chrome }: DeepRequired<ReduxState>) => chrome.user);
-  const navDropdownEnabled = useFlag('platform.chrome.navigation-dropdown');
+  // TODO: remove me!
+  const navDropdownEnabled = true;
+  // TODO: remove me!
```

Comment:
You forgot

Reply:
@karelhala this has to be reverted. We want to feature flag to be in place for now.